openverse_catalog/dags/data_refresh/dag_factory.py

```diff
@@ -135,6 +135,22 @@ def _month_check(dag_id: str, session: SASession = None) -> str:
     )
 
 
+def _month_check_with_reporting(dag_id: str, media_type: str) -> str:
+    """
+    Wrapper for the monthly check function to report which step is starting
+    which step is next to slack.
+    """
+    next_task_id = _month_check(dag_id)
+    next_step = {
+        REFRESH_POPULARITY_METRICS_TASK_ID: "refresh popularity metrics",
```

Comment:
Maybe this one can be `update popularity metrics`, just so that it matches the language of "Popularity metrics update complete" [message when that step completes](https://github.com/WordPress/openverse-catalog/pull/744/files#diff-ea351594f1b0188842f0e6dc9286b8761a7dfdd21aa1b36306c30e0ccef97089R55)? Not a big deal but it makes it slightly easier to see the steps following through.

Reply:
We use `refresh` everywhere else to describe this step, so I'll just update the completion message!